samples/Mvc.Client/Startup.cs

```diff
@@ -17,10 +17,30 @@ public class Startup
     {
         public void ConfigureServices(IServiceCollection services)
         {
-            services.AddAuthentication(options =>
-            {
-                options.SignInScheme = CookieAuthenticationDefaults.AuthenticationScheme;
-            });
+            services.AddAuthentication(CookieAuthenticationDefaults.AuthenticationScheme)
+                .AddCookie(o => {
+                    o.LoginPath = new PathString("/login");
```

Comment:
Remove new PathString and just set it to "/login" and "/signout" there's an implicit conversion from string -> PathString

Reply:
@davidfowl @PinpointTownes There's `PathString`s in `OpenIdAuthenticationOptions` and `OpenIdAuthenticationDefaults`, should I remove them?